client/src/Views/Landing/Login.js

```diff
@@ -47,6 +47,12 @@ class Login extends Component {
     }
   };
 
+  onEnterPress = e => {
```

Comment:
If you wrapped the login and signup in a form, you actually don't need to write this, and some validation will also come for free